grpc-interface-support/test/integration/test_integration_cpp.py

```diff
@@ -26,9 +26,14 @@ def get_subdirs(path: str) -> List[str]:
     return [f.path for f in os.scandir(path) if f.is_dir()]
 
 
-def get_project_cache_dir() -> str:
-    project_caches = os.path.join(os.path.expanduser("~"), ".velocitas", "projects")
-    return get_subdirs(project_caches)[0]
+def get_project_cache_dir(index: int = 0) -> str:
```

Comment:
Any particular reason to have 0 as default here? Order is anyway arbitrary so we do not know what we will get for 0, or?

Reply:
Changed the code to use the new CLI command